insights/combiners/virt_who_conf.py

```diff
@@ -8,18 +8,18 @@
 
     1. command line             # Ignored
     2. environment variables    # Ignored
-    3. /etc/sysconfig/virt-who  # SysconfigVirtWho
-    4. /etc/virt-who.d/*.conf  # VirtWhoConf
+    3. /etc/sysconfig/virt-who  # VirtWhoSysconfig
+    4. /etc/virt-who.d/*.conf   # VirtWhoConf
     5. /etc/virt-who.conf       # VirtWhoConf
 
 """
 
 from insights.core.plugins import combiner
-from insights.parsers.sysconfig.virt_who import SysconfigVirtWho
+from insights.parsers.sysconfig import VirtWhoSysconfig
 from insights.parsers.virt_who_conf import VirtWhoConf
 
 
-@combiner(requires=[SysconfigVirtWho, VirtWhoConf])
+@combiner(requires=[VirtWhoSysconfig, VirtWhoConf])
```

Comment:
@xiangce would you mind to add an examples section to the docs for this combiner?